tpm2/tpm2.go

```diff
@@ -1211,6 +1211,32 @@ func NVDefineSpace(rw io.ReadWriter, owner, handle tpmutil.Handle, ownerAuth, au
 	return err
 }
 
+// NVDefineSpaceNew accepts NVPublic structure and is more flexible.
```

Comment:
Should there be a test added that exercises this new function?

Reply:
Old test covers this.